pymc3/distributions/simulator.py

```diff
@@ -112,10 +112,10 @@ def random(self, point=None, size=None):
         array
         """
         params = draw_values([*self.params], point=point, size=size)
-        if size is None:
+        if not size:
             return self.function(*params)
         else:
-            return np.array([self.function(*params) for _ in range(size)])
+            return np.array([self.function(*params) for _ in range(size[0])])
```

Comment:
I handled `size` this way because now it is passed as a tuple. Ping @aloctavodia to review as this is related to SMC stuff.
Also, test suite passes. :)